container/sagemaker/python_service.py

```diff
@@ -10,6 +10,7 @@
 # distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
 # ANY KIND, either express or implied. See the License for the specific
 # language governing permissions and limitations under the License.
+
```

Comment:
not clear why this exists as a separate module from gunicorn_service

Reply:
I will put 3 falcon apps into 1